src/Servers/IIS/IIS/test/IIS.NewHandler.FunctionalTests/IIS.NewHandler.FunctionalTests.csproj

```diff
@@ -41,7 +38,12 @@
 
   <Target Name="RemoveSourceOverride" BeforeTargets="_CopyOutOfDateSourceItemsToOutputDirectory">
     <ItemGroup>
-      <SourceItemsToRemove Include="@(_SourceItemsToCopyToOutputDirectory)" Condition="$([System.String]::Copy(%(_SourceItemsToCopyToOutputDirectory.FullPath)).Contains('IIS\AspNetCoreModule')) == 'true'" />
+    <!-- There is a race between which dll is copied to the bin directory between ANCM in a package and the built ANCM. 
+        We want to guarantee the ANCM in the nuget package is used, so remove all paths that contain paths in the artifacts directory.
+        -->
+      <SourceItemsToRemove Include="@(_SourceItemsToCopyToOutputDirectory)" Condition="$([System.String]::Copy(%(_SourceItemsToCopyToOutputDirectory.FullPath)).Contains('AspNetCoreModuleShim')) == 'true'" />
```

Comment:
Why do these files end up in _SourceItemsToCopyToOutputDirectory in the first place? Could we prevent that from happening?

Reply:
The reason they are there is that we reference Microsoft.AspNetCore.Server.IntegrationTesting.IIS which includes native assets: https://github.com/aspnet/AspNetCore/blob/master/src/Servers/IIS/IntegrationTesting.IIS/src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj#L30. We also reference `<PackageReference Include="Microsoft.AspNetCore.AspNetCoreModuleV2" Version="2.2.0">` above. These cause both to be copied to
_SourceItemsToCopyToOutputDirectory. I still would like to continue referencing Microsoft.AspNetCore.Server.IntegrationTesting.IIS, maybe I can set a property that causes those components not to be referenced.

Reply:
@natemcmaster I think I came up with a better fix: https://github.com/aspnet/AspNetCore/pull/10858/files#diff-0ec7ab070eb9a3b8d1eac9206a279cddL30